CHANGELOG.md

```diff
@@ -1,22 +1,26 @@
 # Release History
 
-## 1.3.0 (TBD)
+## 1.3.0 (2023-03-28)
 
 ### New Features
 
 - Added support for `delimiters` parameter in `functions.initcap()`.
 - Added support for `functions.hash()` to accept a variable number of input expressions.
-- Added API `Session.RuntimeConfig` for getting/setting/checking the mutability of any runtime configuration.
+- Added API `Session.conf` for getting/setting/checking the mutability of any runtime configuration.
 - Added support managing case sensitivity in `Row` results from `DataFrame.collect` using `case_sensitive` parameter.
```

Comment:
I think instead of "Added support managing case sensitivity..."
it should be:
"Added support for managing case sensitivity..."